scripts/get-test-pass-rate.mjs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT License.
+
+// This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used
+// to get timeline and metrics data so that the test pass rate can be calculated in a later step.
+
+// The Build ID needed to fetch the desired data.
+const BUILD_ID = process.env.BUILD_ID;
+// The token need to make the API calls.
+const ADO_API_TOKEN = process.env.ADO_API_TOKEN;
+
+// The workspace where the new files/folder created in this script will be stored.
+const TEST_WORKSPACE = process.env.TEST_WORKSPACE;
+console.log("BUILD_ID:", BUILD_ID);
+console.log("TEST_WORKSPACE:", TEST_WORKSPACE);
+
+// Create output folder - Note: This requires Node.js fs module
+import * as fs from "fs";
+if (!fs.existsSync(`${TEST_WORKSPACE}/stageFiles`)) {
+        fs.mkdirSync(`${TEST_WORKSPACE}/stageFiles`, { recursive: true });
+        console.log("Folder created");
+}
+const apiUrl = `https://dev.azure.com/fluidframework/internal/_apis/build/builds/${BUILD_ID}/timeline?api-version=7.1-preview.2`;
+
+async function fetchData() {
```

Comment:
Was the extra function necessary? If the file is `.mjs` it should be possible to use `await` at the root of the file, without being inside an `async` function. As is, the fact that we're not awaiting the call to `fetchData` means that the node process might exit immediately without waiting for it to complete. I think it should be possible to do away with the function and just have all the code live "at the root" of the script.